src/realm/util/timestamp_logger.hpp

```diff
@@ -9,15 +9,15 @@
 namespace realm {
 namespace util {
 
-class TimestampStderrLogger : public Logger {
+class TimestampStderrLogger : public StderrLogger {
```

Comment:
It looks like the TimestampStderrLogger is completely unused outside of tests. Can we just remove it?